config/crd/bases/etcd.improbable.io_etcdclusters.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.2.4
+    controller-gen.kubebuilder.io/version: v0.2.2
```

Comment:
Is my `controller-gen` version out of date again? -_-

Reply:
Yes, yes it was. Upgraded to v0.2.4 in 3e13280 or the verify check wouldn't pass.